docs/streaming_how_to.rst

```diff
@@ -15,6 +15,9 @@ by offering a first walk through.  Some features of Tweepy streaming are
 not covered here. See streaming.py in the Tweepy source code. 
 
 API authorization is required to access Twitter streams. 
+
+**Important**: You must authenticate using OAuth 1a if you want to use the streaming API.
```

Comment:
```suggestion
**Important**: You must authenticate using _OAuth 1a_ to get a token with _user_ context if you want to use the streaming API. This can be done using an App Access Token as on Getting Started page, or using the longer user-login flow.
```